ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/ByteCodeBuilder.java

```diff
@@ -0,0 +1,174 @@
+package org.hyperledger.besu.ethereum.core;
+
+import kotlin.UInt;
```

Comment:
using a Kotlin class?

Reply:
Good catch! It was auto-imported by mistake.